general/CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ list(APPEND SRCS
   globals.cpp
   hash.cpp
   isockstream.cpp
+  jit/jit.cpp
```

Comment:
How is `jit/parser.cpp` handled?

Reply:
As it is an executable required before the compilation of the declared JIT source files, it is handled in the `makefile` or the top `CMakeLists.txt` directly.